docs/distribution/README.md

```diff
@@ -0,0 +1,222 @@
+# OCI Distribution
+
+We introduce new REST APIs in the registry to support storing signature objects together with the target artifacts, and retrieving them for verification.
+
+## Table of contents
+1. [Sample APIs for retrieving signatures](#Sample-APIs-for-retrieving-signatures)
+2. [Sample APIs for storing signatures](#Sample-APIs-for-storing-signatures)
+3. [Implementation](#Implementation)
+
+## Sample APIs for retrieving signatures
+
+The list of signatures for an artifact can be retrieved from the registry.
+
+### _Get a list of paginated signatures by tag_
+
+### Request
+`GET http://localhost:5000/v2/hello-world/manifests/v1.0/signatures/`
+
+### Response
+
+```json
+{
+    "tag": "v1.0",
+    "digest": "sha256:80559bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042",

```

Comment:
we've been getting feedback that we should always show full OCI Descriptors
- mediaType (index or manifest)
- digest
- size

This would apply to the header info, confirming what was retrieved, and the signatures collection returned. Each signature would be a descriptor, not just the digest.